lib/core-server/src/utils/StoryIndexGenerator.ts

```diff
@@ -138,6 +147,24 @@ export class StoryIndexGenerator {
     // Pull out each file's stories into a list of stories, to be composed and sorted
     const storiesList = await this.ensureExtracted();
 
+    // Compute other imports
+    const allImports = storiesList.reduce((acc, entry) => {
```

Comment:
```suggestion
    const storyImports = storiesList.reduce((acc, entry) => {
```

Calling it `allImports` is confusing. Originally we store *all* imports that the csf file uses on the `story` object. Then we make this has of all `importPath`s that *are* story files. So I would argue it isn't "all imports" but a crucial subset.